doc/classes/Viewport.xml

```diff
@@ -306,6 +306,8 @@
 			If [code]true[/code], the viewport should render its background as transparent.
 		</member>
 		<member name="use_debanding" type="bool" setter="set_use_debanding" getter="is_using_debanding" default="false">
+			If [code]true[/code], uses a fast post-processing filter to make banding significantly less visible in 3D. 2D rendering is [i]not[/i] affected by debanding. See also [member rendering/anti_aliasing/quality/use_debanding].
```

Comment:
The correct link here should be [member ProjectSettings.rendering/anti_aliasing/quality/use_debanding]